java/src/com/liferay/nativity/control/win/WindowsNativityUtil.java

```diff
@@ -65,15 +65,17 @@ private static void _load() {
 
 			_loaded = true;
 
+		    System.out.println("Successfully loaded " + nativityDllName);
```

Comment:
What do you mean there's work involved to make this right?  Can't we just add this back in if it ever stops working?  I would still probably remove it because its a print line that we'll have every time we run our app even if the logger is off, but its up to you.